### PR TITLE
More cdda

### PR DESCRIPTION
**Pull request by privatezero**
A couple more CDDA additions - let's see how many typos I can slip through while half the crew is at NTTW3

### CODE REVIEW

**Review by kieranjol — APPROVED** [review]
I can't verify the commands but all looks great to me, brilliant additions.

**Comment by privatezero** [comment]
Left out offset info with `Cdda2wav` as it seems to only support sector offset - not sample offset. Also didn't add XLD CLI as it is a hollow shell (no pun intended...ok maybe a little bit intended) of the GUI in terms of functionality.

**Comment by ablwr** [comment]
Too many of us traveling after NTTW to review your PR, Weav!!!

**Review by ablwr — APPROVED** [review]
I TRUST YOU !

File: index.html [hunk 1]
```diff
@@ -2404,13 +2404,49 @@ <h2 id="similar-tools">Similar tools: tips &amp; tricks 🎩🐰</h2>
...
+      </div>
+      <!-- Find Drive Offset for Exact CD Ripping -->
+
+      <!-- Rip with CD Paranoia -->
+        <label class="recipe" for="cdparanoia">Rip a CD with CD Paranoia</label>
+        <input type="checkbox" id="cdparanoia">
+        <div class="hiding">
+          <h3>Rip a CD with CD Paranoia</h3>
+          <p><code>cdparanoia -L -B -O <em>[Drive Offset]</em> <em>[Starting Track Number]</em>-<em>[Ending Track Number]</em> <em>output_file.wav</em></code></p>
+          <p>This command will use CD Paranoia to rip a CD into separate tracks while compensating for the sample offset of the CD drive. (For more information about drive offset see <a href=#find-offset>the related ffmprovisr command.</a>)</p>
```
Comment (retokromer) [inline index.html]: I guess, the addresses should always be in quotation marks: `href="#find-offset"`.

**Review by retokromer — COMMENTED** [review]
One nit, otherwise LGTM.

File: index.html (hunk 1 above)
Comment (retokromer) [reply index.html]: I am actually sure.

**Review by retokromer — APPROVED** [review]

**Pull request merged**